public/translations/pt-BR.json

```diff
@@ -406,6 +406,8 @@
     "my_transfers": "Minhas transferências",
     "no_one": "Ninguém",
     "edit_success": "Perfil salvo com sucesso",
+    "locations": "Localizações",
```

Comment:
although I know it is on the Figma design to use plural, it really don't make any sense. I think it should be location. 

any thoughts @amiskov @RodrigoLeiteF?

Reply:
> Figma design to use a plural

It looks like a typo. Profile and Profile Edit pages have singular and that label on the Public Profile is the same:
![image](https://user-images.githubusercontent.com/140053/82337754-c9818500-99f4-11ea-9dad-7f7ba286a2a8.png)